apps/web/lib/settings/member-table.tsx

```
@@ -59,18 +59,20 @@ export const MemberTable = ({ members }: { members: OT_Member[] }) => {
 				...activeTeamRef.current,
-				memberIds: updatedMemberIds
+				managerIds: updatedManagerIds
 			});
 		} else {
 			// Remove manager
-			const updatedMemberIds = [...new Set([...currentManagers, employeeId])];
+			const updatedManagerIds = currentManagers.filter(id => id !== employeeId);
+
+			return updateOrganizationTeam(activeTeamRef.current, {
+				...activeTeamRef.current,
+				managerIds: updatedManagerIds,
```

Biome

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)